prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -475,7 +475,7 @@ abstract class CoolTest {
     suspend fun pollForStepResult(
         reportId: ReportId,
         taskAction: TaskAction,
-        maxPollSecs: Int = 840,
+        maxPollSecs: Int = 120,
```

Comment:
So we've bumped this up in order to get the tests pass more reliably, can we leave this as is?

Reply:
Actually with the automatic retries, reducing the timeout makes sense to me